apko.yaml

```diff
@@ -1,34 +1,31 @@
 package:
   name: apko
   version: "0.25.4"
-  epoch: 0
+  epoch: 1
   description: Build OCI images using APK directly without Dockerfile
   copyright:
     - license: Apache-2.0
   dependencies:
     runtime:
       - ca-certificates-bundle
 
-environment:
-  contents:
-    packages:
-      - build-base
-      - busybox
-      - ca-certificates-bundle
-      - go
-
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/chainguard-dev/apko
       tag: v${{package.version}}
       expected-commit: 352a64da25397ab97b18c107f1e3c3df2a709c62
 
-  - runs: |
-      make apko
-      install -m755 -D ./apko "${{targets.contextdir}}"/usr/bin/apko
-
-  - uses: strip
+  - uses: go/build
+    with:
+      output: apko
+      packages: .
+      ldflags: |
+        -buildid= \
+        -X sigs.k8s.io/release-utils/version.gitVersion=$(git describe --dirty --tags --always) \
+        -X sigs.k8s.io/release-utils/version.gitCommit=$(git rev-parse HEAD) \
+        -X sigs.k8s.io/release-utils/version.gitTreeState=$(shell git diff --quiet >/dev/null 2>&1 || echo "dirty" || echo "clean") \
+        -X sigs.k8s.io/release-utils/version.buildDate=$(date -u +'%Y-%m-%dT%H:%M:%SZ')
```

Comment:
need a double check if these ldflags are correct.
ref: https://github.com/chainguard-dev/apko/blob/59634991a9fe36c31c3e8ccffbed415e3bd71f96/Makefile#L35

Reply:
to validate ldflags are intact in b/w updates we can grep for ${{package.version}} while checking version.

Reply:
done, thanks!

Reply:
`apko version | grep ${{package.version}}`